keyboards/bm60rgb_iso/info.json

```
@@ -0,0 +1,77 @@
+{
+    "keyboard_name": "BM60 RGB ISO",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"K00 (B0,D0)", "x":0, "y":0},
+                {"label":"K01 (B0,D1)", "x":1, "y":0},
+                {"label":"K02 (B0,D2)", "x":2, "y":0},
+                {"label":"K03 (B0,D3)", "x":3, "y":0},
+                {"label":"K04 (B0,D5)", "x":4, "y":0},
+                {"label":"K05 (B0,D4)", "x":5, "y":0},
+                {"label":"K06 (B0,D6)", "x":6, "y":0},
+                {"label":"K07 (B0,D7)", "x":7, "y":0},
+                {"label":"K08 (B0,B4)", "x":8, "y":0},
+                {"label":"K09 (B0,B5)", "x":9, "y":0},
+                {"label":"K0A (B0,B6)", "x":10, "y":0},
+                {"label":"K0B (B0,C6)", "x":11, "y":0},
+                {"label":"K0C (B0,C7)", "x":12, "y":0},
+                {"label":"K0D (B0,F7)", "x":13, "y":0, "w":2},
+                {"label":"K10 (B1,D0)", "x":0, "y":1, "w":1.5},
+                {"label":"K11 (B1,D1)", "x":1.5, "y":1},
+                {"label":"K12 (B1,D2)", "x":2.5, "y":1},
+                {"label":"K13 (B1,D3)", "x":3.5, "y":1},
+                {"label":"K14 (B1,D5)", "x":4.5, "y":1},
+                {"label":"K15 (B1,D4)", "x":5.5, "y":1},
+                {"label":"K16 (B1,D6)", "x":6.5, "y":1},
+                {"label":"K17 (B1,D7)", "x":7.5, "y":1},
+                {"label":"K18 (B1,B4)", "x":8.5, "y":1},
+                {"label":"K19 (B1,B5)", "x":9.5, "y":1},
+                {"label":"K1A (B1,B6)", "x":10.5, "y":1},
+                {"label":"K1B (B1,C6)", "x":11.5, "y":1},
+                {"label":"K1C (B1,C7)", "x":12.5, "y":1},
+                {"label":"K1D (B1,F7)", "x":13.75, "y":1, "w":1.25, "h":2},
+                {"label":"K20 (B2,D0)", "x":0, "y":2, "w":1.75},
+                {"label":"K22 (B2,D2)", "x":1.75, "y":2},
+                {"label":"K23 (B2,D3)", "x":2.75, "y":2},
+                {"label":"K24 (B2,D5)", "x":3.75, "y":2},
+                {"label":"K25 (B2,D4)", "x":4.75, "y":2},
+                {"label":"K26 (B2,D6)", "x":5.75, "y":2},
+                {"label":"K27 (B2,D7)", "x":6.75, "y":2},
+                {"label":"K28 (B2,B4)", "x":7.75, "y":2},
+                {"label":"K29 (B2,B5)", "x":8.75, "y":2},
+                {"label":"K2A (B2,B6)", "x":9.75, "y":2},
+                {"label":"K2B (B2,C6)", "x":10.75, "y":2},
+                {"label":"K2C (B2,C7)", "x":11.75, "y":2},
+                {"label":"K2D (B2,F7)", "x":12.75, "y":2},
```

Moving ISO Enter's object into place.

```suggestion
                {"label":"K20 (B2,D0)", "x":0, "y":2, "w":1.75},
                {"label":"K22 (B2,D2)", "x":1.75, "y":2},
                {"label":"K23 (B2,D3)", "x":2.75, "y":2},
                {"label":"K24 (B2,D5)", "x":3.75, "y":2},
                {"label":"K25 (B2,D4)", "x":4.75, "y":2},
                {"label":"K26 (B2,D6)", "x":5.75, "y":2},
                {"label":"K27 (B2,D7)", "x":6.75, "y":2},
                {"label":"K28 (B2,B4)", "x":7.75, "y":2},
                {"label":"K29 (B2,B5)", "x":8.75, "y":2},
                {"label":"K2A (B2,B6)", "x":9.75, "y":2},
                {"label":"K2B (B2,C6)", "x":10.75, "y":2},
                {"label":"K2C (B2,C7)", "x":11.75, "y":2},
                {"label":"K2D (B2,F7)", "x":12.75, "y":2},
                {"label":"K1D (B1,F7)", "x":13.75, "y":1, "w":1.25, "h":2},
```